### DISCUSSION TITLE
update to 2.4.1

### DISCUSSION THREAD

**Discussion by Mav1814** [Security-Onion-Solutions/securityonion#10320] [2.4] [answered]
Hi, 

I hope everyone is okay!!

I have a doubt, to upgrade from SO beta 1 to beta 2 (2.4.1), do I have to do a fresh installation?

cheers and thanks in advance

**Comment by InfosecGoon** [comment, marked as answer]
Yes, you have to do a fresh install -- there's no soup capability in the beta releases.

**Comment by Mav1814** [comment]
thank you for the quick answer @InfosecGoon. Cheers
